README.md

```diff
@@ -15,8 +15,8 @@
 [s6]: https://tokei.rs/b1/github/amethyst/laminar?category=code
 [s7]: https://codecov.io/gh/amethyst/laminar/branch/master/graphs/badge.svg
 
-Laminar is a semi-reliable UDP-based protocol for multiplayer games. This library implements wrappers around the UDP-protocol,
-and provides a lightweight, message-based interface which provides certain guarantees like reliability and ordering.
+Laminar is an application-level transport protocol that provides configurable reliability and ordering guarantees build on top of UDP. 
```

Comment:
Laminar is an application-level transport protocol which provides configurable reliability and ordering guarantees built on top of UDP.